src/network/networkd-route.c

```diff
@@ -581,6 +581,37 @@ static bool route_type_is_reject(const Route *route) {
         return IN_SET(route->type, RTN_UNREACHABLE, RTN_PROHIBIT, RTN_BLACKHOLE, RTN_THROW);
 }
 
+static void log_route_debug(const Route *route, const char *prefix, const Link *link) {
```

Comment:
nit: given it's about network routes, when I read "prefix" I thought this variable was about something completely different. Maybe rename to "log_prefix" or something like that?

Reply:
Haha, renamed to `str`.